docs/developer/decision-records/2022-06-20-mvd-onboarding/README.md

```diff
@@ -0,0 +1,36 @@
+# Participants onboarding in MVD
+
+This document outlines the infrastructure steps for the scenario of onboarding participants to the Minimum Viable Dataspace (MVD).
+
+## Precondition - GAIA-X membership
+
+To join the MVD, each participant must have a Verifiable Credential (VC) signed by GAIA-X Authority which proves GAIA-X membership.
+
+At present the MVD deployment pipeline does not interact with the real GAIA-X Authority. A simulated GAIA-X Authority is deployed within MVD for demonstration purposes.
+
+## Onboarding process - GAIA-X membership verification
+
+During the onboarding process, the GAIA-X membership claim is verified by the Dataspace Authority.
+The onboarding flow is presented in the [distributed authorisation sub-flow document](https://github.com/agera-edc/MinimumViableDataspace/tree/feature/20-rs-adr-target/docs/developer/decision-records/2022-06-16-distributed-authorization).
+Actors in the onboarding scenario:
+
+- _Participant A_ is a putative participant that wants to join the MVD.
+- _Participant B_ is the Dataspace Authority (Registration Service and its `did:web` document).
+- _Authority_ is the simulated GAIA-X Authority.
+
+## Infrastructure
+
+To enable above precondition and onboarding scenario in MVD, the following steps need to be implemented in the MVD infrastructure: 
+
+1. The MVD deployment workflow will generate the private and public keys for GAIA-X Authority.
+2. The MVD deployment workflow will deploy the GAIA-X Authority DID document containing the public key.
+3. The GAIA-X private key will be used in the participant deployment workflow to generate GAIA-X membership Verifiable Credentials. Additionally, 
+   it will be published as Github artifact so that it can be used locally to onboard additional participants.
+4. A CLI client for IdentityHub will be implemented. It will be used in the MVD deployment workflow to populate the participant's Identity Hub with GAIA-X 
```

Comment:
nit: We are using "will" or "will be" here in multiple places. I understand that right now we do not have all the items listed here that is why maybe we are proposing to do this in future. But at some point, these items will be implemented and then this ADR will not reflect correct state of the MVD.

So maybe we can just list out necessary steps and not say that this will be implemented. For example:

_IdentityHub CLI client used in the MVD deployment workflow to populate the participant's Identity Hub with GAIA-X_

Reply:
Can be ignore if you do not agree. Fine for me. Thanks.